src/StatusQ/Components/StatusMemberListItem.qml

```diff
@@ -0,0 +1,64 @@
+import QtQuick 2.0
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+
+StatusListItem {
+    id: root
+
+    property string nickName: ""
+    property string userName: ""
+    property string keyChat: ""
+    property bool isMutualContact: false
+    property var trustIndicator: StatusMemberListItem.TrustedType.None
+    property string imageSource: ""
+    property bool isOnline: false
+
+    enum TrustedType {
+        None, //0
+        Verified, //1
+        Untrustworthy //2
+    }
+
+    // Subtitle composition:
+    function printSubtitile() {
+        var print = ""
+        if(root.userName !== "")
+            print = "(" + root.userName + ")"
+
+        if(print !== "" && root.keyChat !== "")
+            // Composition
+            print += " • " + printShortKeyChat(root.keyChat)
```

Comment:
Can we rename this? Maybe I not used to the print naming, but it could be rather compose SubTitile? anyway, just my feeling.

Reply:
> Just a thought here: we also have `StatusAddress` which can be used to render an address. Although it might make things unnecessarily complex here.
> 
> Have a look if you want, you call!

Nice component `StatusAddress`! Thanks for sharing it @PascalPrecht!

Next time I’ll consider it for my implementation. In this case, I should redesign the component itself to use it so I’ll keep it as it is, if you don’t mind.

Reply:
> Can we rename this? Maybe I not used to the print naming, but it could be rather compose SubTitile? anyway, just my feeling.

Sure! I'll rename it!